gp-populate-anything/gppa-populate-all-results.php

```diff
@@ -10,10 +10,13 @@
  * CSS Class Name setting.
  */
 add_filter( 'gppa_process_template_value', function( $template_value, $field, $template_name, $populate, $object, $object_type, $objects, $template ) {
-	if ( strpos( $field->cssClass, 'gppa-populate-all' ) !== false ) {
-		$values = array();
+	if ( ! empty( $objects ) && strpos( $field->cssClass, 'gppa-populate-all' ) !== false ) {
+		$values = array( $template_value );
 		foreach ( $objects as $_object ) {
-			$values[] = $object_type->get_object_prop_value( $_object, $template );
+			if ( $_object === $object ) {
+				continue;
+			}
```

Comment:
Just performance. I'd actually prefer to remove since performance isn't a concern. With this, it skips the single object that was selected for population since we already have its parsed template. Only lingering concern here was if the `$object` passed by the filter would always be the first object in the `$objects` array.

Reply:
Ahh, got it. FWIW there's a boat load of caching around process_template so if something has been processed already you'll get a cached version of it.